### PR TITLE
Intro - The Twelve Days of Dancer

### PR DESCRIPTION
**Pull request by cromedome**
Here's the kickoff article.  Some questions for you all:

* I wasn't sure about putting D1 on notice. I don't think this is being unrealistic at all. While it is not fun and festive, if I am discussing the state of Dancer (ALL of Dancer), it seems the responsible thing to do. Penny for your thoughts?

* GitHub discussions - not something we have discussed, Core Team, and something that was completely off of my radar until earlier today, actually. I can nix it, but I think it's worth asking the community what they want.

* Thoughts on my mini Hacktoberfest rant? Was trying to express frustration in a slightly humorous way. I may have failed.

Thanks for your prompt feedback. This is the first article, I need to get it merged soon!

### CODE REVIEW

**Comment by GeekRuthie** [comment]
In order of your post, my thoughts, for whatever they are worth:

- It's important news, so needs to be said. And no, it's not holly-jolly, but 2020 is like that. You did fine here.

- As a non-Core dev, and a GitLab partisan, I don't have any *firm* opinion on this, but my gut says that getting the discussion in the same space as the code is a Good Thing.

- You did not fail.  If you need help next year weeding the obvious spam PRs, I'd be happy to help.

**Comment by cromedome** [comment]
Thanks @GeekRuthie!

**Review by bigpresh — APPROVED** [review]

**Review by SysPete — APPROVED** [review]
1 typo spotted but otherwise :+1:. As for your questions:

- :+1: 
- Mentioning this seems reasonable, though I'm not sure whether I want yet another place to have to visit.
- :+1:

File: pending/13-the-twelve-days-of-dancer.pod [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+=item * Hacktoberfest
+
+Dear Reader, if only I could express the level of stress and frustration I
+encountered the first days of Hacktoberfest this year - the number of spam
+PRs we received was pretty ridiculous, and we are a much smaller project than
+some on GitHub, and I cannot imagine how they coped with the onslaught of 
+spammy PRs, all for a lousy t-shirt (ok, the shirt was pretty cool...).
+
+To Digital Ocean's credit, I think they did an admirable job of trying to 
+stop the bleeding. But for me, the spirit of Hacktoberfesr has been ruined,
```
[the remark this answers is not in the document]
Comment (cromedome) [reply pending/13-the-twelve-days-of-dancer.pod]: Fixed, thanks!

**Pull request merged**